frontend/.gitignore

```diff
@@ -31,4 +31,5 @@ coverage
 
 test-results/
 playwright-report/
+yarn.lock
```

Comment:
The yarn.lock should be considered in the commits. Please, take out this modification